books/free-programming-books-sr.md

```diff
@@ -0,0 +1,8 @@
+### Index
+
+* [C](#c)
+
+
+### C
+
+* [Beej-ov vodič za mrežno programiranje - Korištenje Internet soket-a](https://web.archive.org/web/20181008134854/http://users.teol.net:80/~mvlado/sockets/) - B. Hall (HTML)
```

Comment:
New files should have its entry at README.md

Locale is OK. Serbian: https://www.localeplanet.com/icu/sr/index.html